backend/routes/events.js

```diff
@@ -71,7 +71,35 @@ router.get('/admin/all', async (req, res, next) => {
     } catch (err) {
         handleError(err, req, res, next);
     }
+});
+
+//Get all events  (non-admin)
+router.get('/non_admin/dashboard', async (req, res, next) => {
+    try {
+        if (!req.user) {
+            throw new Error('401__You must be logged in');
+        } else {
+            const userId = processInput(req.user.a_id || req.user.v_id || req.user.f_id, "idNum", "user id");
```

Comment:
no need to the code line82 (const userId ...) to line90
we don't need to precessInput the volunteerId which is req.user.v_id because it's coming from the database and for sure it's a numerical id